apps/frontend/src/layout/AuthLayout.tsx

```diff
@@ -9,6 +11,10 @@ import Logo from "../assets/images/logo.png";
 const clientId = import.meta.env.VITE_GOOGLE_CLIENT_ID;
 
 export function AuthLayout() {
+    const { logout } = useAuth();
+    useEffect(() => {
+        logout();
```

Comment:
@AfeezGL What does this do?

Reply:
It makes sure the cache is purged before the user logs in. I've had instances where I logged in to another workspace but I was seeing surveys for the last workspace I used.

Reply:
@jdamilola Can you land on this page with a valid session?

Reply:
@jdamilola No.

So how this issue happens is if your cookies get cleared for some reasons and you try to reload the page the `AuthProvider` redirects you to the login screen [here](https://github.com/IntegraflowHQ/integraflow/blob/1446133f2677c5e66bc833be43ef74caadf588d5/apps/frontend/src/modules/auth/AuthProvider.tsx#L522) because you don't have a valid session. When you login and go to the survey list page, you'll see the surveys of the last workspace you used because the cache wasn't purged.
I think what I'm trying to do here can be done in the `AuthProvider` just before redirecting the user to the login screen.